src/dialogs/more-info/controls/more-info-vacuum.ts

```diff
@@ -303,6 +302,7 @@ class MoreInfoVacuum extends LitElement {
               "battery_level",
               Math.round(stateObj.attributes.battery_level)
             )}
+            %
```

Comment:
This is handled by `formatEntityAttributeValue` https://github.com/home-assistant/frontend/blob/dev/src/data/entity_attributes.ts#L71